languages/csharp/exercises/concept/nullability/.meta/config.json

```
@@ -0,0 +1,8 @@
+{
+  "contributors": [
```

We've recently renamed this to `authors`:

```suggestion
  "authors": [
```